src/stravalib/model.py

```diff
@@ -599,11 +674,20 @@ class AthleteStats(strava_model.ActivityStats):
 
 
 class MetaAthlete(strava_model.MetaAthlete, BoundClientEntity):
```

Comment:
How would you document this class (the meta classes in general @jsamoocha ) if we don't talk about BoundClient. Should we focus on what is inheretied from strava model (so the x attributes this object carries?) (in this case just the ID?

Reply:
I added a few commits with suggested changes to the docstrings in our Meta classes.

Reply:
thank you!